src/styles/blocks/card.css

```diff
@@ -0,0 +1,97 @@
+.card {
+  width: 200px;
+  min-height: 400px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 19px 16px;
+}
+
+.card__img {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto ;
+}
+
+.card__title {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 11px;
+  line-height: 18px;
+  margin: 40px 0 0;
+  color: #060b35;
+}
+
+.card__code {
+  font-family: "Roboto", sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin: 4px 0 0;
+}
+
+.card__revievs {
+  display: flex;
+  margin: 16px 0 0;
```

Comment:
try to use only `margin-bottom`

if some tags have default margins, you can reset it for tags